cli/cmd/proxy/read/command.go

```diff
@@ -208,9 +227,9 @@ func (c *ReadCommand) validateFlags() error {
 	return nil
 }
 
-// filtersPassed returns true if the user has passed a flag which filters the
-// output.
-func (c *ReadCommand) filtersPassed() bool {
+// tableFiltersPassed returns true if a flag which filters the
+// set of tables output is passed.
+func (c *ReadCommand) tableFiltersPassed() bool {
```

Comment:
I went with `areTablesFiltered` because it doesn't have the confusion of "passed", it uses the verb form for booleans, and it's the clearest I could come up with.

Reply:
Thanks!